backend/utils.py

```diff
@@ -92,7 +92,13 @@ def call_github_action(
         if params:
             payload["inputs"] = params
 
-    response = post(url, headers=headers, data=json.dumps(payload))
+    response: Response = post(url, headers=headers, data=json.dumps(payload))
+    if response.status_code >= 400:
+        raise RuntimeError(
+            f"Error calling GitHub action {action_name} with params:"
```

Comment:
## The fix
The `GH_LIMITED_PERSONAL_ACCESS_TOKEN` variable was missing from `.env`. This is the result of Joe's computer dying in October, and Joe having used Siggie's `.env` since then. At some point, the `.env` of TermHub was updated to match this, and at that point is when these GH action calls stopped working.